src/store/filter/filter.store.ts

```diff
@@ -33,6 +33,7 @@ export class FilterStore {
   private _attributeNameToAdd: string = ''
   private _presetModifiedState: PresetModifiedState =
     PresetModifiedState.NotPreset
+  private _isChanging = false
```

Comment:
In form development it is common to name such properties 'isDirty'. When the form is touched (some value is enterd or changed) it is considered to be 'dirty'. Or simply 'isTouched'.
I propose to call it 'isFilterTouched'

Reply:
Done